modules/ex/src/clj/exoscale/ex.clj

```diff
@@ -272,7 +274,13 @@
           (~sym ~msg ~data nil))
          ([~msg ~data ~cause]
           (let [~data (assoc ~data :type ~type)]
-            (ex-info ~msg ~type ~data ~cause)))))))
+            (ex-info ~msg ~type ~data ~cause))))
+
+       (defn ~bangsym
+         ~(format "Builds an exception with %s and throws it." sym)
+         [& ~args]
+         (throw (apply ~sym ~args))))))
```

Comment:
`(throw (~sym ~@args))`, we can skip the apply at runtime.

Reply:
Right, adapted accordingly